types/wlr_output.c

```diff
@@ -93,6 +93,8 @@ static void output_bind(struct wl_client *wl_client, void *data,
 	send_current_mode(resource);
 	send_scale(resource);
 	send_done(resource);
+
+	wl_signal_emit(&output->events.bind, resource);
```

Comment:
I thought the intention for new code is to either use a dedicated struct or NULL for signals?

Reply:
Changed.

Reply:
We can use `wl_resource`, but only if we're 100% sure that we'll never need to extend the API again. It's always safer to just introduce a new struct.